// error line: 4

Main (->)
    let a <- 1 then ()
end
